jacktrip.pro

```diff
@@ -56,7 +63,7 @@ INCLUDEPATH += faust-src-lair/stk
   weakjack {
     message(Building with weak linking of JACK)
     INCLUDEPATH += externals/weakjack
-    DEFINES += USE_WEAK_JACK
+    DEFINES += USE_WEAK_JACK NDEBUG
```

Comment:
This looks like if we build with weakjack we also disable all asserts. I think somebody should open an issue for Windows Terminal Preview instead of this.